### ISSUE TITLE
Add support to use browser extensions (Chrome & firefox at least)

### ISSUE THREAD

**Issue by javiergarciavera** [cypress-io/cypress#16703]
Sometimes, we need to automate scenarios that includes browser extensions, and this is not supported at the moment by cypress (as far as I know), so the workaround is using puppeteer in parallel. So I was wondering if it's possible to add it in Cypress so it would make it easy and that would make Cypress even greater.

**Comment by pavelstudeny** [comment]
There is a few plugins: https://www.npmjs.com/search?q=cypress-browser-extension-plugin

**Comment by MrRahulR** [comment]
> There is a few plugins: https://www.npmjs.com/search?q=cypress-browser-extension-plugin

Actually, the plugin seems dead now, ☠ it's been a long it's not maintained and also it supports the older cypress version.
I was using it in the past but I see right now it has a lot of PR open but the repo isn't active. ❌

**Comment by MrRahulR** [comment]
> There is a few plugins: https://www.npmjs.com/search?q=cypress-browser-extension-plugin

Actually, the plugin seems dead now, ☠ 

The [repository](https://github.com/ejoubaud/cypress-browser-extension-plugin) for the plugin - it's been a long it's not maintained and also it supports the older cypress version.
I was using it in the past but I see right now it has a lot of PR open but the repo isn't active. ❌

**Comment by flotwig** [comment]
For now, using CDP or Puppeteer in Cypress to automate this is the way. Closing as a duplicate of #14808.

**Comment by Depetrol** [comment]
@flotwig can you give some specifications on using puppeteer for browser extension? I've been trying but cannot seem to connect to CDP
